lom/src/pluginmgr/pluginmgr_main.go

```diff
@@ -23,8 +30,8 @@ import (
 *        ./LoMpluginMgr -proc_id="proc_0" -syslog_level=5
 *
 * Environment Variable :
-*       LOM_TESTMODE_NAME - LoM run mode
-*               - Optional (default: "no")
```

Comment:
Can we have/update README with this info, rather than inside one file. This affects globally everyone

Reply:
added new Readme and included common info. This Readme is for specific plugin manager info